utils.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import numpy as np
+from PIL import Image
+import os
+
+
+def load_params(params_path: str) -> dict:
+    
+    ac_params = {}
+
+    with open(params_path) as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=';')
+
+        keys = []
+        values = []
+
+        row_number = 0
+
+        for row in csv_reader:
+            if row_number == 0:
+                keys = row
+            elif row_number == 1:
+                aux = []
+                for i in row:
+                    try:
+                        aux.append(float(i))
+                    except ValueError:
+                        aux.append(i)
+                values = aux
+            else:
+                break
+            row_number += 1
+
+        ac_params = dict(zip(keys, values))
+
+    return ac_params
+
+        
+def get_coords_from_csv(filepath):
```

Comment:
Must be generalized:

idea:

coords = {roi_i : { contour_j : { x : [xCoords], y : [yCoords]}}}